### PR TITLE
refactor(widgetbook): :recycle: adjust content

### PR DESCRIPTION
**Pull request by jenshor**
- improves Readme content for `widgetbook` package

### CODE REVIEW

**Comment by jenshor** [comment]
@Roaa94 check out the rendered Readme here: https://github.com/widgetbook/widgetbook/tree/refactor/pubspec/packages/widgetbook

File: packages/widgetbook/README.md [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+- 🔧 **Maintained** by a fast growing Flutter startup from Germany.
+- 👩🏼‍💻 **Support** via our [Discord](https://discord.com/invite/zT4AMStAJA) community.
 
+Widgetbook can do much more
 
+- 📙 **Catalogue existing Widgets**. Never develop Widgets twice. 
+- 📦 **Develop in isolation**. Don't think about business logic. 
+- 🎛 **Knobs**. Define how your team interacts with your Widgets by defining customizable knobs. 
+- ⚡️ **Hot reload**. Quickly find and fix bugs.
+- ✨ **Beautiful documentation**. Readable, easy to understand and ever-improving.
```
Comment (Roaa94) [inline packages/widgetbook/README.md:29]: Do you think it makes sense to add a link to the documentation here?

File: packages/widgetbook/README.md [hunk 2]
```diff
@@ -1,23 +1,44 @@
...
 
+# Widgetbook Cloud
 
+To simplify the review process of professional teams, we provide our collaboration platform Widgetbook Cloud
+- 🌍 **Share your Widgets** with designers, product owners, clients, etc.
+- 📃 **Simplified reviews** with our guided process. 
+- 🔗 **Figma Integration**. Connect your Widgets to their design counterpart in Figma.
```
Comment (Roaa94) [inline packages/widgetbook/README.md:36]: Maybe here also a link to the website or demo call?

**Review by Roaa94 — APPROVED** [review]
This looks great 🎉
Just left a couple off comments.

File: packages/widgetbook/README.md (hunk 1 above)
Comment (jenshor) [reply packages/widgetbook/README.md:29]: Thought about it. Most people will look at the readme via pub.dev which has a documentation button in the sidebar. Do you think it's necessary to put a link here?

Comment (Roaa94) [reply packages/widgetbook/README.md:29]: Hmm I see your point. Still a link in this context might be helpful sometimes in my opinion if there is no harm in adding it.

Comment (jenshor) [reply packages/widgetbook/README.md]: Yup. Added it.

File: packages/widgetbook/README.md (hunk 2 above)
Comment (jenshor) [reply packages/widgetbook/README.md]: Yup. Added it.

**Pull request merged**